packages/firestore/test/unit/local/local_store.test.ts

```diff
@@ -244,14 +235,28 @@ class LocalStoreTester {
     return this;
   }
 
+  toNotContainIfEager(
```

Comment:
Nice! I would maybe flip this to `toContainIfLru`, since that feels like the stricter assert. But that's just my personal opinion :)